Plandomizer.py

```diff
@@ -523,35 +523,52 @@ def set_shuffled_entrances(self, worlds, entrance_pools, target_entrance_pools,
     def fill_bosses(self, world, prize_locs, prizepool):
         count = 0
         used_items = []
-        for (name, record) in pattern_dict_items(self.locations):
+        locations = {}
+        if self.locations:
+            locations = {loc: self.locations[loc] for loc in random.sample(sorted(self.locations), len(self.locations))}
```

Comment:
There is no reason to do this. Plando makers should place the locations in the order they wish.

Reply:
So... yo want to remove it?